clients/erlang/README.md

```diff
@@ -0,0 +1,22 @@
+<h1>Download dependencies and build</h1>
```

Comment:
please, no inline HTML. use Markdown markup.

The link to rebar's homepage/install guide would be awesome (saves people from googling). Also short *how to* get erlang development environment would be nice (again less googling, also for us to try this out). Check python's client readme for an example: https://github.com/futurice/space-tyckiting/blob/master/clients/python/README.md